docs/admin/projects.rst

```diff
@@ -677,9 +677,8 @@ Default value can be changed by :setting:`COMMIT_PENDING_HOURS`.
 Lock on error
 +++++++++++++
 
-Enables locking the component on repository error (failed pull, push or merge).
-Locking in this situation avoids adding another conflict which would have to be
-resolved manually.
+Locks a component upon any repository error (failed pull, push or merge).
```

Comment:
```suggestion
Locks the component upon any repository error (failed pull, push or merge).
```

Reply:
I think "a" is more hypothetical since this isn't basic functionality and needs to be turned on?

Reply:
> I think "a" is more hypothetical since this isn't basic functionality and needs to be turned on?

It describes a setting which is available for a specific component. This setting locks exactly the component, in which the setting is located in... 

`a` or `any chosen component`, suggests that there is another setting to set the component. Or it's a drop-down list selection... it just sounds off-place here IMHO...

Reply:
please close the discussion once you come to an end with it.